cobigen-plugins/cobigen-javaplugin-parent/cobigen-javaplugin/src/main/java/com/devonfw/cobigen/javaplugin/merger/JavaMerger.java

```diff
@@ -134,6 +134,7 @@ public String merge(File base, String patch, String targetCharset) throws MergeE
    */
   private String addGeneratedAnnotation(JavaClass baseClass, JavaClass parsedAnnotationClass, String lineDelimiter) {
 
+    final String ANNOTATION_IMPORT = "javax.annotation.Generated";
```

Comment:
Should be a class variable instead. And maybe rename to: GENERATED_ANNOTATION_CLASS